specification/agfood/data-plane/Microsoft.AgFoodPlatform/preview/2021-03-31-preview/agfood.json

```diff
@@ -5603,47 +5654,55 @@
           }
         },
         "x-ms-examples": {
-          "OAuthTokens_Delete": {
-            "$ref": "./examples/OAuthTokens_Delete.json"
+          "OAuthTokens_GetCascadeDeleteJobDetails": {
+            "$ref": "./examples/OAuthTokens_GetCascadeDeleteJobDetails.json"
           }
         }
-      }
-    },
-    "/oauth/tokens/:connect": {
-      "post": {
+      },
+      "put": {
         "tags": [
           "OAuthTokens"
         ],
-        "description": "Returns Connection link needed in the OAuth flow.",
-        "operationId": "OAuthTokens_GetOAuthConnectionLink",
-        "consumes": [
-          "application/json"
-        ],
+        "description": "Create remove job for OAuth token.",
+        "operationId": "OAuthTokens_CreateCascadeDeleteJob",
         "produces": [
           "application/json"
         ],
         "parameters": [
+          {
+            "in": "path",
+            "name": "jobId",
+            "description": "Job Id supplied by end user.",
+            "required": true,
+            "type": "string"
+          },
           {
             "in": "query",
-            "name": "api-version",
-            "description": "The requested API version",
+            "name": "farmerId",
+            "description": "Id of the farmer.",
```

Comment:
Please capitalize `ID` in the descriptions.

Reply:
Done.